app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,39 @@
+/*
+  Place all the styles related to the matching controller here.
+  They will automatically be included in application.css.
+*/
+
+.comments-form {
+  margin-top: -25px;
+  margin-bottom: 70px;
+}
+
+.comments {
+  margin: 30px 0 70px 0;
+  background: #EBEBEB;
```

Comment:
Color literals like `#EBEBEB` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#EBEBEB` should be written as `#ebebeb`